Added Shadow Role section for each events-team and Activities to Project Management events team (#4017)
kubernetes/community: events/events-team/registration/README.md

  comfortable with queries and syntax is incredibly helpful, both for for the
  registration approval process and later report generation.

## Shadow to the Registration Lead Role

> This role can include a shadow role as well. The Registration Lead can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow to the Registration Lead is responsible for handling all Registration lead related activities in the absence of the Lead. 

Reviewer: ```suggestion
This role can include one or more shadows. The Registration Lead can delegate/assign the tasks to the shadows. The expectation from a shadow is to lead one of the events-team roles in an upcoming summit. The shadows to the Registration Lead are responsible for handling all Registration lead related activities in the absence of the Lead. 
```

More of the same 👍
